plugins/pluginmanagers/lz-n.nix

```
@@ -146,6 +149,42 @@ lib.nixvim.plugins.mkNeovimPlugin {
       };
     in
     {
+      keymaps = mkOption {
+        type = types.listOf (mkMapOptionSubmodule {
+          extraOptions.plugin = mkOption {
+            type = with types; either str lzPluginType;
+            example = "telescope.nvim";
+            description = ''
+              The name of the plugin to lazy-load when the keymap is triggered.
+              This must match the name used in the `plugins` list or be a valid plugin name that can be loaded.
+            '';
+          };
+        });
+        default = [ ];
+        example = [
+          {
+            plugin = "telescope.nvim";
+            key = "<leader>ff";
+            action = nestedLiteralLua "function() require('telescope.builtin').find_files() end";
+            options = {
+              desc = "Find files";
+            };
```

`toPretty` will render is as you have it regardless, but the in-code value may be neater as:

```suggestion
            options.desc = "Find files";
```

?